src/mento.ts

```diff
@@ -320,4 +331,13 @@ export class Mento {
     )
     return exchanges[0]
   }
+
+  /**
+   * Returns a new Mento instance connected to the given signer or provider
+   * Similar to contract.connect() in Ethers.js
+   * @returns new Mento instance
+   */
+  connect(signerOrProvider: Signer | providers.Provider) {
+    return new Mento(this.broker.address, signerOrProvider, this.exchanges)
+  }
```

Comment:
nit: can we move this a bit to the top, maybe after the `create` methods? 

also, will this method ever be used for connecting a provider? Since it does `this.broker.address` it already assumes that the existing instance has a provider instead, so I wondering if `connectSigner` or something makes more sense.